### PR TITLE
added solution HW 30-03-1 

### PR DESCRIPTION
**Pull request by interami**

### CODE REVIEW

**Comment by interami** [comment]
is everything ok?

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 1]
```diff
@@ -9,12 +9,16 @@
...
      * (пр. сыграли 1:2, а ставил 0:3 - то пользователь у выиграше)
      * то возвращать 1
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        return (result1 == bet1) && (result2 == bet2)
+                ? 2 : (result1 - result2) * (bet1 - bet2) > 0
+                ? 1 : (result1 - result2) == 0 && (bet1 - bet2) == 0
+                ? 1 : 0;
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FootballMatch.java]: please don't use arithmetical operations (+ - / *) for this task

**Comment by interami** [comment]
удалил mvn в описании задания

**Review by Tarabarnik — COMMENTED** [review]
sorry, the review didn't send immediately

File: src/main/java/core/basesyntax/FootballMatch.java [hunk 2]
```diff
@@ -15,6 +15,10 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        return (result1 == bet1) && (result2 == bet2)
+                ? 2 : ((result1 > result2 && bet1 > bet2)
+                 || (result1 < result2 && bet1 < bet2)
+                 || (result1 == result2 && bet1 == bet2))
+                 ? 1 : 0;
```
Comment (Tarabarnik) [inline src/main/java/core/basesyntax/FootballMatch.java]:
```suggestion
        return (result1 == bet1) && (result2 == bet2) ? 2 : 
        (result1 > result2 && bet1 > bet2) || 
        (result1 < result2 && bet1 < bet2) || 
        (result1 == result2 && bet1 == bet2) ? 1 : 0;
```
much prettier, isn't it?

**Review by Tarabarnik — APPROVED** [review]

**Comment by Tarabarnik** [comment]
your code has checkstyle issues. please, solve them

**Review by Tarabarnik — APPROVED** [review]

**Pull request closed without merging**